parts/linux/cloud-init/artifacts/cse_start.sh

```diff
@@ -85,7 +85,7 @@ upload_logs() {
     # fall back to WALA collector
     if test -x /opt/azure/containers/aks-log-collector.sh; then
         # Call AKS Log Collector
-        /opt/azure/containers/aks-log-collector.sh
+        /opt/azure/containers/aks-log-collector.sh >/dev/null 2>&1
```

Comment:
Dumping it to /dev/null makes it really hard to troubleshoot any problems later - would it be better to dump it to /var/log/azure/aks/cse-aks-log-collector.log or something similar?

Reply:
I"m in favour of keeping the traceability.  We might even want to grep in the future of common error in that log file